conan/cli/commands/graph.py

```diff
@@ -397,13 +398,14 @@ def _find_in_remotes(conan_api, dict_nodes, remotes):
             try:
                 remote_ref_list = conan_api.list.select(ref_pattern, package_query=None,
                                                         remote=remote)
-                if not remote_ref_list.recipes:
-                    continue
-            except ConanException as message:
-                ConanOutput().warning(f"{message} in remote {remote.name}")
+            except ConanConnectionError as error:
```

Comment:
This is not fully correct. There are other exceptions like ``AuthenticationException, \
    ForbiddenException, RequestErrorException`` that are not a ConnectionError, and it still should be considered an error, not silently skipped. It is only when the server succesfully returns a 404-not found that we can continue